x-kiosk: Update Application

munoz0raul opened the pull request:
- Changed based image to debian:bullseye-slim
- Updated the USER id to weston (63).

mike-scott commented on x-kiosk/Dockerfile:
```diff
@@ -1,9 +1,36 @@
-  lightdm
+# For testing in the LmP
+# docker build --tag hub.foundries.io/lmp/x-kiosk:latest .
+# docker run -it --rm -v /tmp/.X11-unix:/tmp/.X11-unix -v /var/run/dbus:/var/dbus hub.foundries.io/lmp/x-kiosk:latest www.foundries.io
```
Let's fix this line to:
```
# docker run -it --rm -v /tmp/.X11-unix:/tmp/.X11-unix --user "63" hub.foundries.io/lmp/x-kiosk:latest www.foundries.io
```

munoz0raul replied:
done!

mike-scott reviewed (changes requested):
Looks good except for the testing commands.  Can you verify my change still works?

mike-scott commented:
And we should squash the 2 commits into 1.

munoz0raul commented:
@mike-scott, I updated the command to what I have used and squashed the commits. Builds are falling, didn't see if was working before. Do you know if these actions are up-to-date?

mike-scott reviewed (approved):
LGTM

mike-scott commented:
ci test
